src/dotnet/Chat/TextEntryAttachmentExt.cs

```diff
@@ -3,5 +3,5 @@ namespace ActualChat.Chat;
 public static class TextEntryAttachmentExt
 {
     public static bool IsImage(this TextEntryAttachment attachment)
-        => attachment.ContentType.OrdinalIgnoreCaseStartsWith("image");
+        => attachment.Media.ContentType.OrdinalIgnoreCaseStartsWith("image");
```

Comment:
one more usage for potential attachment.IsImage() ext method

Reply:
It's the `IsImage` ext method )